CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog for `passwdgen`
 
+## `v0.3.2` - 22 April 2023
```

Comment:
Given that this change involves a toolchain upgrade, is it not worth releasing it as v0.4.0?

Reply:
6062673